Containerfile

```diff
@@ -46,20 +40,13 @@ RUN .github/scripts/compile.sh
 
 FROM registry.access.redhat.com/ubi8/ubi:latest
 
-RUN dnf -y --setopt=tsflags=nodocs update \
- && dnf -y --setopt=tsflags=nodocs install \
-      http://mirror.centos.org/centos/8-stream/BaseOS/x86_64/os/Packages/centos-gpg-keys-8-4.el8.noarch.rpm \
-      http://mirror.centos.org/centos/8-stream/BaseOS/x86_64/os/Packages/centos-stream-repos-8-4.el8.noarch.rpm \
- && dnf config-manager --set-enabled powertools \
- && dnf clean all
-
-# gdb and valgrind are part of final image as they can be used as debug options for Skupper
-RUN dnf -y --setopt=tsflags=nodocs install \
+# gdb and sanitizers are part of final image as they can be used as debug options for Skupper
+RUN dnf -y --setopt=install_weak_deps=False --setopt=tsflags=nodocs install \
     glibc \
     cyrus-sasl-lib cyrus-sasl-plain cyrus-sasl-gssapi libuuid openssl \
     python3 \
     libnghttp2 \
-    gdb valgrind \
```

Comment:
@jiridanek is proposing the removal of valgrind, so that we can get
rid of the CentOS 8 repos, we must also remove it as a valid debug option.
@grs Are you ok with that?

Reply:
Actually, replaced valgrind; instead of `valgrind skrouterd` I propose you do `skrouterd_asan`. That is instrumented binary running with the address sanitizer. The Containerfile builds the router twice, first to get skrouterd, then to get skrouterd_asan binary.

Reply:
Yes, fine with me, but we should update the skupper command as well. At present it enforces only valgrind or gdb as valid options. We can change that to whatever is recommended, just make it a boolean or even remove the option entirely and provide some other way to enable it.

Reply:
was the valgrind debug option part of Red Hat's TP3? I don't recall seeing valgrind in the image there, but I might remember wrong. Was valgrind and gdb expected to be available in the Red Hat's GA?